.github/workflows/ci.yml

```diff
@@ -1,16 +1,25 @@
 name: Elixir CI
 
-on: push
+on:
+  pull_request:
+  push:
+      branches:
+      - master
 
 jobs:
-  build:
+  mix_test:
+    name: mix test (OTP ${{matrix.otp}} | Elixir ${{matrix.elixir}})
+    strategy:
+      matrix:
+        otp: [21.x, 22.x]
+        elixir: [1.8.x, 1.9.x, 1.10.x]
```

Comment:
Can you please list those pairs explicitly? Otherwise the matrix can get really large and it becomes harder to manage when we some Elixir versions do not support some OTP releases. Thanks!

Reply:
Done in https://github.com/elixir-ecto/myxql/pull/114/commits/8557f7a18e1fa336496cf0ab188828b6eb9b7994